### ISSUE TITLE
Crash when selecting subreddit other than frontpage and all

### ISSUE THREAD

**Issue by Tunous** [ccrama/Slide#107]
## Info
Test for #103 without xposed. As you can see it still crashed.
## Stacktrace
```java.lang.NullPointerException
at me.ccrama.redditslide.Activities.SubredditOverview.doSubSidebar(SubredditOverview.java:305)
at me.ccrama.redditslide.Activities.SubredditOverview$OverviewPagerAdapter$1.onPageSelected(SubredditOverview.java:1298)
at android.support.v4.view.ViewPager.disp
atchOnPageSelected(ViewPager.java:1788)
at android.support.v4.view.ViewPager.scrollToItem(ViewPager.java:569)
at android.support.v4.view.ViewPager.setCurrentItemInternal(ViewPager.java:553)
at android.support.v4.view.ViewPager.setCurrentItemInternal(ViewPager.java:514)
at android.support.v4.view.ViewPager.setCurrentItem(ViewPager.java:495)
at android.support.design.widget.TabLayout$ViewPagerOnTabSelectedListener.onTabSelected(TabLayout.java:1627)
at android.support.design.widget.TabLayout.selectTab(TabLayout.java:837)
at android.support.design.widget.TabLayout.selectTab(TabLayout.java:809)
at android.support.design.widget.TabLayout$Tab.select(TabLayout.java:1077)
at android.support.design.widget.TabLayout$1.onClick(TabLayout.java:643)
at android.view.View.performClick(View.java:4278)
at android.view.View$PerformClick.run(View.java:17429)
at android.os.Handler.handleCallback(Handler.java:725)
at android.os.Handler.disp
atchMessage(Handler.java:92)
at android.os.Looper.loop(Looper.java:137)
at android.app.ActivityThread.main(ActivityThread.java:5099)
at java.lang.reflect.Method.invokeN
ative(N
ative Method)
at java.lang.reflect.Method.invoke(Method.java:511)
at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:803)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:570)
at dalvik.system.N
ativeStart.main(N
ative Method)```
Reported via Slide Crash Capture

**Comment by Tunous** [comment]
I guess it's this line: https://github.com/ccrama/Slide/blob/7d664ad7922262358337b74e5263f1a3c3daad26/app/src/main/java/me/ccrama/redditslide/Activities/SubredditOverview.java#L305

**Comment by ccrama** [comment]
Ah, are you in single view? If so, I have fixed that with 824b092117ae42d3759f65f86ab984530f3d0965

**Comment by ccrama** [comment]
Merged with #103

**Comment by Tunous** [comment]
It's in multi view on the latest alpha. Happens when scrolling to next subreddit. 

Would help if you used tags for each release so it's easy to keep track :)

Thanks for this awesome Reddit app btw.

**Comment by ccrama** [comment]
Probably fixed in the update I published a few minutes ago (#7)!

And thank you :)

**Comment by Tunous** [comment]
Good job. Fixed.
